activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -650,6 +658,10 @@ def remove_references(*args)
       def foreign_key(*args) # :nodoc:
         @base.add_foreign_key(name, *args)
       end
+      
+      def foreign_key_exists?(*args) # :nodoc:
```

Comment:
@claudiob I think it was due to the fact that `foreign_key` had a no-doc as well. It was previously only used by `references`. Should be fine to document them though.